### PR TITLE
remove oc dependency on origin/pkg/quota/apis/quota

### PR DESCRIPTION
**Pull request by sanchezl**
Replaced dependency on `github.com/openshift/origin/pkg/quota/apis/quota` with `github.com/openshift/oc/pkg/helpers/quota`.

### CODE REVIEW

File: pkg/oc/lib/describe/describer.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ import (
 	buildapihelpers "github.com/openshift/oc/pkg/helpers/build"
 	ocbuildapihelpers "github.com/openshift/oc/pkg/helpers/build"
 	routedisplayhelpers "github.com/openshift/oc/pkg/helpers/route"
+	quotahelpers "github.com/openshift/origin/staging/src/github.com/openshift/oc/pkg/helpers/quota"
```
Comment (deads2k) [inline pkg/oc/lib/describe/describer.go]: use the vendored dep, so `github.com/openshift/oc/pkg/helpers/quota`

**Comment by deads2k** [comment]
If verify passes, then there's another direct dep on this.  Try to clear a few more.

File: pkg/oc/lib/describe/describer.go (hunk 1 above)
Comment (sanchezl) [reply pkg/oc/lib/describe/describer.go]: Done

**Pull request merged**